src/components/SecondaryNavDisclosure.vue

```diff
@@ -0,0 +1,59 @@
+<template>
```

Comment:
@MeganSteenkamp Suggestion: Refer to these as Sidebar disclosures. "Nav" can be misinterpreted for the navbar along the top of the page.